### DISCUSSION TITLE
Formily 如何使用 element-ui 组件

### DISCUSSION THREAD

**Discussion by strayover** [alibaba/formily#2135] [Q&A 2.x] [answered]
新人小白，第一次学习使用Formily，在vue2.0下安装 element-ui 以及 @formily/core、@formily/vue 
把官方文档例子写入时修改 引入input 组件，import { Input } from 'element-ui'，界面可以显示但是input无法输入，不清楚是否会有其他配置项，还请大神帮忙，另希望寻求vue2.0 + Formily + element-ui demo以便学习参考
![image](https://user-images.githubusercontent.com/24351955/132783273-d267600b-dd1b-4b24-93c5-75933483711f.png)

**Comment by muuyao** [comment, marked as answer]
见 https://element.formilyjs.org/

**Reply by strayover** [reply]
谢谢，
